### PR TITLE
refactor(util): COMMANDS_GUILD_ID constant arg order

### PR DESCRIPTION
**Pull request by itohatweb**
**Please describe the changes this PR makes and why it should be merged:**

**Status**

- [x] Code changes have been tested against the Discord API, or there are no
  code changes

**Semantic versioning classification:**

- [x] This PR changes the library's interface (methods or parameters added)
  - [ ] This PR includes breaking changes (methods removed or renamed,
    parameters moved or removed)
- [ ] This PR **only** includes non-code changes, like changes to documentation,
  README, etc.

### CODE REVIEW

File: src/api/handlers/webhook.ts [hunk 1]
```diff
@@ -214,7 +216,18 @@ export function createSlashCommand(options: CreateSlashCommandOptions) {
...
+/** Fetchs the global command for the given ID. If a guildID is provided, the guild command will be fetched. */
+export async function getSlashCommand(commandID: string, guildID?: string) {
+  const result = await RequestManager.get(
+    guildID
+      ? endpoints.COMMANDS_GUILD_ID(applicationID, guildID, commandID)
+      : endpoints.COMMANDS_ID(applicationID, commandID),
+  );
+
+  return result as SlashCommand;
+}
```
Comment (ayntee) [inline src/api/handlers/webhook.ts:228]: This should have a separate PR.

**Comment by ayntee** [comment]
image

**Comment by itohatweb** [comment]
Resolved

**Review by ayntee — APPROVED** [review]
LGTM

**Pull request merged**